package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "app",
+  "private": "true",
+  "dependencies": {
+    "@hotwired/stimulus": "=3.2.2",
+    "@hotwired/turbo-rails": "=8.0.0-beta.1",
+    "@rails/webpacker": "5.4.4",
+    "axios": "=1.6.2",
+    "react": "=18.2.0",
+    "react-dom": "=18.2.0",
+    "react-redux": "=9.0.2",
+    "react-router-dom": "=6.20.1",
+    "stylelint-csstree-validator": "^3.0.0",
+    "webpack": "=5.89.0",
+    "webpack-cli": "=5.1.4",
+    "webpack-dev-server": "=4.15.1"
+  },
```

Comment:
- [ ] @Zuwairanajma Great work implementing the changes requested, however, you removed some dependency `redux toolkit` which you are using in your redux store leading to an error when building the react app using yar build / npm run build.
`hint, used the code below deleted your package-lock.json, then ran npm install and npm run build and the app worked without errors`
`{
  "name": "app",
  "private": "true",
  "dependencies": {
    "@hotwired/stimulus": "^3.2.2",
    "@hotwired/turbo-rails": "^8.0.0-beta.1",
    "@reduxjs/toolkit": "^1.9.7",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "axios": "^1.5.1",
    "react-redux": "^8.1.3",
    "react-router-dom": "^6.20.0",
    "webpack": "^5.89.0",
    "webpack-cli": "^5.1.4"
  },
  "scripts": {
    "build": "webpack --config webpack.config.js"
  },
  "devDependencies": {
    "@babel/core": "^7.23.3",
    "@babel/preset-env": "^7.23.3",
    "@babel/preset-react": "^7.23.3",
    "babel-loader": "^9.1.3",
    "json-loader": "^0.5.7"
  }
}`